app/javascript/graphql-explorer/components/graphQLFetcher.js

```diff
@@ -0,0 +1,24 @@
+// Defines a GraphQL fetcher using the fetch API. You're not required to
+// use fetch, and could instead implement graphQLFetcher however you like,
+// as long as it returns a Promise or Observable.
+export default (graphQLParams) => {
+  // This example expects a GraphQL server at the path /graphql.
+  // Change this to point wherever you host your GraphQL server.
+  return fetch('/graphql', {
+    method: 'post',
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'X-Auth-Token': localStorage.miq_token
```

Comment:
What is this, really? Is this the token from the cookie or something?

Reply:
`localStorage` is your browser's local storage.
(In Chrome, right click -> `Inspect`. Checkout the `Application` tab that shows the local storage)

After a successful login in the UI, the auth token is stored in `localStorage.miq_token`, which is then used by the API calls or in this case, the `fetch` call.